experimental/packages/opentelemetry-sdk-node/src/types.ts

```diff
@@ -40,6 +40,7 @@ export interface NodeSDKConfiguration {
   instrumentations: (Instrumentation | Instrumentation[])[];
   resource: IResource;
   resourceDetectors: Array<Detector | DetectorSync>;
+  mergeResourceWithDefaults: boolean;
```

Comment:
Showing my typescript-compat inexperience here: Does this need to be an optional property of the interface?

Ah, or perhaps not because `Partial<NodeSDKConfiguration>` is being used for this interface.

Reply:
you're correct, this would indeed be a breaking change because we export the interface. I was under the impression that we'd just export the partial type.

I'll just mark it as optional for now.

```suggestion
  mergeResourceWithDefaults?: boolean;
```